app/views/categories/index.html.erb

```diff
@@ -10,7 +10,7 @@
                 <p><%= category.description %></p>
             </div>
             <div class="c2">
-                <%= link_to "Start", '#', class: 'btn btn-primary float-right' %>
+                <%= link_to "Start", lessons_path(category_id: category.id, user_id: current_user.id), method: :post, class: 'btn btn-primary float-right' %>
```

Comment:
@CharlotteDS No need to pass `user_id: current_user.id`, since you can access `current_user` in the controller